gradle/spek.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "org.junit.platform.gradle.plugin"
```

Comment:
I don't actually have any tests right now (woot! woot!) so I technically don't need to commit this file.

I did push some spek and kotlin dependencies up to `spinnaker-dependencies` to cut down on duplication between projects.